README.md

```diff
@@ -454,6 +454,37 @@ AS confederation parameters. Valid values are String, keyword 'default'.
 ##### `enforce_first_as`
 Enable/Disable enforces the neighbor autonomous system to be the first AS number listed in the AS path attribute for eBGP. Valid values are 'true', 'false', and 'default'.
 
+##### `event_history_cli`
+Enable/Disable cli event history buffer. Valid values are 'true', 'false', and 'default'.
+
+##### `event_history_cli_size`
+Set cli event history buffer size. Valid values are 'small', 'medium', 'large', 'disable' or 'default'.
```

Comment:
My understanding is that a property can be both boolean and string. how to use one property to differentiate between no event-history cli and event-history cli disabled.
 So I use two properties and refer to default_originate in cisco_neighbor_af.